Sources/ApolloCodegenLib/ApolloCodegen.swift

```diff
@@ -56,12 +62,14 @@ public class ApolloCodegen {
       rootURL: rootURL
     )
 
+    try validate(config: configContext)
```

Comment:
I've separated these two `validate` calls because the first one includes path syntax validation and should be done before compilation of the schema + operations. The second validation ensures no schema name conflict.

I don't like that they're now separate, some ideas to improve:
* rename the second one from `validate` to something else
* move the schema name logic somewhere else